src/engine/duel/ai/common.asm

```diff
@@ -391,16 +391,26 @@ PickTwoAttachedEnergyCards:
 	ld a, $ff
 	ret
 
-; copies $ff terminated buffer from hl to de
-CopyBuffer:
+; copies an $ff-terminated list from hl to de
+; preserves bc
+; input:
+;	hl = address from which to start copying the data
+;	de = where to copy the data
+CopyListWithFFTerminatorFromHLToDE_Bank8:
 	ld a, [hli]
 	ld [de], a
 	cp $ff
 	ret z
 	inc de
-	jr CopyBuffer
+	jr CopyListWithFFTerminatorFromHLToDE_Bank8
 
-; zeroes a bytes starting at hl
+; zeroes a bytes starting from hl.
+; this function is identical to 'ClearNBytesFromHL' in Bank $2,
+; as well as ClearMemory_Bank5' and 'ClearMemory_Bank6'.
```

Comment:
The copy in bank 6 is currently called `ClearMemory`, not `ClearMemory_Bank6`. I also wouldn't be opposed to renaming `ClearNBytesFromHL` to `ClearMemory_Bank2`.

Reply:
Sorry about that. That must have been a label change from own disassembly that I missed. Just to be consistent, I could change the function label to match the comment, along with the change to `ClearNBytesFromHL` that you suggested.

Reply:
Actually, it appears that the comment was correct. I changed `ClearMemory` to `ClearMemory_Bank6` in my last pull request, but I can still change `ClearNBytesFromHL` to `ClearMemory_Bank2` to be consistent with the other three.

Reply:
Ooops, sorry I was behind.